### ISSUE TITLE
delta station operating tables are invisible

### ISSUE THREAD

**Issue by jtroopa** [tgstation/tgstation#52915]
Issue reported from  Round ID: [144187](https://scrubby.melonmesa.com/round/144187) (/tg/Station Manuel [ENGLISH] [US-WEST] [ROLEPLAY] [99% LAG FREE])

Reporting client version: 513.1507

Exactly what it says. Operating tables on delta station are invisible and can't be interacted with, while still blocking the tile.

**Comment by brotherbeyondd** [comment]
And on meta...

**Comment by luxidoptera** [comment]
This is happening on MetaStation, too. I think I know why it's happening- I'm admin on a private server, and I tried deleting and replacing the operating table, and I noticed something- the operating table isn't just vanishing, it's _linking to the metal tables next to it,_ and since there are no sprites for that for obvious reasons, the op table becomes invisible and intangible.

I imagine a typo in the code caused the operating tables to be considered "normal" tables for the purposes of sprite linking.

![image](https://user-images.githubusercontent.com/15697751/90163227-d0a20000-dd63-11ea-89b3-86273bd342cb.png)
